packages/fxa-react/package.json

```diff
@@ -61,15 +61,15 @@
     "eslint": "^6.8.0",
     "file-loader": "4.2.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^25.3.0",
+    "jest": "^24.9.0",
```

Comment:
jest downgrade, too?

Reply:
yeah, "standardizing" on what `react-scripts` is pinned to